greenpithumb/moisture_sensor.py

```diff
@@ -1,5 +1,31 @@
+import adc
+
+#placeholder values
+_MIN_MOISTURE_LEVEL = 0.0
+_MAX_MOISTURE_LEVEL = 100.0
```

Comment:
I think the range is 0 to 1023, based on this
https://github.com/dicksondickson/PlantFriends/blob/master/sensor_node/RFM69/sensor_node.ino#L23-31